etc/helm/pachyderm/templates/NOTES.txt

```diff
@@ -10,8 +10,10 @@ pachctl config set active-context local
 pachctl port-forward
 {{- end -}}
 
-{{ if and (.Values.oidc.mockIDP) (.Values.pachd.enterpriseLicenseKey) (.Values.pachd.activateAuth) }}
+{{ if not (include "oidc.hasRealIDP" .) }}
```

Comment:
I think this would be simpler if we don't add `oidc.hasRealIDP` and instead include `not .Values.oidc.upstreamIDPs`  here.

Reply:
Yeah - that's simpler and makes the relationship between real and mock IDPs clear